frontend/src/modules/dataset/dataset-browser.utils.tsx

```diff
@@ -161,7 +161,7 @@ function tablePropertiesForScopes (state: DatasetBrowserState, updateDatasetCont
             cell (item) {
                 return (
                     <Link data-cy={item.name} onFollow={() => {
-                        updateDatasetContext({ type: item.type }, '', false);
+                        updateDatasetContext({ ...state.datasetContext, type: item.type }, '', false);
```

Comment:
If nothing else in this file was changed why is this needed?

Reply:
Hmm I was trying to get the browsers/selectors working, I think I just missed them in reverting my changes